jbpm-flow-builder/src/main/java/org/jbpm/compiler/xml/XmlProcessReader.java

```diff
@@ -101,4 +117,19 @@ void setProcesses(final List<Process> processes) {
     public ProcessBuildData getProcessBuildData() {
         return (ProcessBuildData) this.parser.getData();
     }
+
+    protected String processParserMessage(Object parent, Attributes attr, String errorMessage) {
+        String nodeId = (attr == null  || attr.getValue("id") == null) ? "" : attr.getValue("id");
+        String nodeName = (attr == null  || attr.getValue("name") == null) ? "" : attr.getValue("name");
+
+        if(parent != null && parent instanceof RuleFlowProcess) {
```

Comment:
@krisv yes. there is no reliable way in ExtensibleXmlParser to traverse up to find the top level parent atm. I can work on adding that in but maybe best to create a new jira for it. the info the jira requested to show would not be meaningful if the parent node was a container node like subprocess for example. wdyt?